ecosystem/typescript/sdk_v2/src/crypto/ed25519.ts

```diff
@@ -160,7 +160,13 @@ export class Ed25519PrivateKey extends PrivateKey {
     return new Ed25519PrivateKey({ hexInput: keyPair.secretKey.slice(0, Ed25519PrivateKey.LENGTH) });
   }
 
-  publicKey(): PublicKey {
+
+  /**
+   * Derive the Ed25519PublicKey for this private key.
+   * 
+   * @returns Ed25519PublicKey
+   */
+  publicKey(): Ed25519PublicKey {
```

Comment:
Isnt this change from a different pr?

Reply:
Ahh yes! Interesting, this was created using Graphite as a stacked PR, I guess it did not handle that.